flac.md

```diff
@@ -449,6 +449,8 @@ Directly after the last metadata block, one or more frames follow. Each frame co
 
 Each frame header stores the audio sample rate, number of bits per sample and number of channels independently of the streaminfo metadata block and other frame headers. This was done to permit multicasting of FLAC files but it also allows these properties to change mid-stream. Because not all environments in which FLAC decoders are used are able to cope with changes to these parameters during playback, a decoder MAY choose to stop decoding on such a change. A decoder that does not check for such a change could be vulnerable to buffer overflows. See also [the section on security considerations](#security-considerations).
 
+Note that storing audio with changing audio parameters in FLAC results in various practical problems. For example, these changes of audio parameters must happen on a frame boundary, or the process will not be lossless. When a variable blocksize is chosen to accomodate this, note that blocks smaller than 16 samples are not allowed and it is therefore not possible to store an audio stream in which these parameters change within 16 samples of the last change or the start of the file. Also, since the streaminfo metadata block can only accomodate a single set of parameters, it is only valid for part of such an the audio stream. Instead, it is RECOMMENDED to store an audio stream with changing parameters in FLAC encapsulated in a container capable of handling such changes, as these do not suffer from the mentioned limitations. See [section container mappings](#container-mappings) for details.
+
```

Comment:
`accomodate` > `accommodate`

Reply:
I just noticed I use audio parameters as well as audio properties. I think I should change this to audio properties throughout the document. Any thoughts?

Reply:
Indeed, it’s a good idea to unify.